docs/API/Results/ResultInterface.md

```
@@ -136,18 +136,18 @@ Execute on `Success`, mutate the result, and continue the chain.
 
 
 ```php
-public function unwrap(R $default = NULL): R|T
+public function unwrap(?callable $fn = NULL): mixed
 ```
 
-Return the unwrapped value of a `Success`, or a default value.
+Return the unwrapped value of a `Success`, or throws the error of a `Failure`. When a callable is provided, it is called with the value of the `Success` or `Failure`, and its return value is returned.
 
 #### Parameters
 | Name | Type | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)